index.js

```diff
@@ -11,6 +11,32 @@ const hexToRGB = (hexColor) => {
   return [R, G, B];
 };
 
+const rgbToRGB = (rgbColor) => {
+  [R, G, B] = rgbColor
+    .replace(/[^\d,]/g, "")
+    .split(",")
+    .map((el) => parseInt(el));
```

Comment:
Here you can go with something bit more descriptive than `el`.
`RGBValue` would be an option